docs/security.md

```diff
@@ -0,0 +1,184 @@
+| Read-only access to [deployments](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-deployments)                                                                                                                           | The app needs to read deployments to support sending deployment events to Jira. This is permission is only used when you are using [GitHub deployments](https://github.com/atlassian/github-for-jira/blob/main/docs/deployments.md).                                                                                                                                                                                        
+| Read-only access to metadata                                                                                                                                                                                                                                         | All GitHub apps have read-only metadata permission set by default. This is a [mandatory requirement by GitHub](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#metadata-permissions) and is needed to provide access to a collection of read-only endpoints with metadata for various resources. These endpoints do not provide sensitive private repository information.                                             
+| Read and write access to [issues](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-issues) and [pull requests](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-pull-requests) | The app reads new issues and pull requests to check them for Jira issue keys. If there is a key, it will make them visible in the corresponding Jira issue. The app also replaces valid Jira issue keys in issue and pull request descriptions with a link to the corresponding Jira issue (“unfurling”). For this, the app requires write access. |
+| Read-only access to [members](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-members)                                                                                                                                   | The needs read access to the members of your GitHub organizations to determine whether they are an admin of the org or not. Only admins are allowed to change the configuration of the GitHub for Jira app.                                                                                                                                   |
+
+The GitHub for Jira app has `READ`, `WRITE`, and `DELETE` permissions on the Jira API. The permissions to the Jira API cannot be finer grained, at the moment, so that these permissions are scoped across the whole Jira API.
```

Comment:
I was wondering if we should use "The" when we are referring to the app by name (here and in other places)...

Reply:
Yeah, I think the "the" is appropriate